Tests/DAO/UserDAOTest.swift

```diff
@@ -25,21 +25,47 @@ class UserDAOTest: XCTestCase {
 		
 		userDAO = UserDAO(fileStorage: fileStorage)
 		userDAO.clearSession()
+		userDAO.replaceUserContexts(identities: [])
 	}
 	
 	override func tearDown() {
 		userDAO.clearSession()
 	}
 	
-	func testSetUserContext() {
-		let identityContext = IdentityContextMessage(analyticsKey: "key1") {
-			$0.userId = "userId1"
-		}
+	func testAddUserContext() {
+		let identityContext = IdentityContext(analyticsKey: "key1")
 		
-		userDAO.setUserContext(identity: identityContext)
-		let persistedIdentityContext = userDAO.getUserContext()
+		userDAO.addUserContext(identity: identityContext)
+		let persistedIdentityContext = userDAO.getUserContexts().last
+		assert(userDAO.getUserContexts().count == 1)
```

Comment:
Why `assert`? Is not better to use the XCTest functions like `XCTAssertEquals` for that?

This will stop executing the rest of the test if something fails

Reply:
Yeah, more clean to read the logs. 👍